modules/appeals_api/config/schemas/v2/200996.json

```diff
@@ -80,17 +80,22 @@
       "type": "object",
       "properties": {
         "name":  { "$ref": "#/definitions/hlrCreateRepresentativeName" },
-        "phone": { "$ref": "#/definitions/hlrCreatePhone" }
+        "phone": { "$ref": "#/definitions/hlrCreatePhone" },
+        "emailAddressText": {
+          "type": "string",
+          "format": "email",
+          "maxLength": 100
+        }
       },
       "additionalProperties": false,
       "required": [ "name", "phone" ]
     },
 
     "hlrCreateRepresentativeName": {
-      "type": "string",
+      "type": "object",
       "properties": {
-        "firstName": { "type": "string"},
-        "lastName":  { "type": "string"}
+        "firstName": { "type": "string", "maxLength": 30 },
```

Comment:
These limits are smaller than the first and last name fields fit, but the ticket asks that we ensure the full name will fit in the rep signature box if/when we support the rep's digital signature.

Reply:
Without concern for the signature, the limits are 40 for first name and 60 for last.

Reply:
Even with autosizing the limits are that small inside the signature? Boo :(